src/AutoMapper.Net4/AutoMapper.Net4.csproj

```diff
@@ -49,8 +49,446 @@
     <Reference Include="System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089, processorArchitecture=MSIL" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\AutoMapper\**\*.cs">
-      <Link>%(RecursiveDir)%(Filename)%(Extension)</Link>

```

Comment:
Yea it always does that.  You know this.  I thought since it's a PR of a PR it will get fixed after merge to development.  Didn't even bother with this.